manim/mobject/coordinate_systems.py

```diff
@@ -495,3 +503,321 @@ def get_coordinate_labels(self, *numbers, **kwargs):
+
+              .. manim:: PolarPlanePIradians
+                    :ref_classes: PolarPlane
+                    :save_last_frame:
+
+                    polarplot = PolarPlane(azimuth_units="PI radians").scale(0.4)
+                    polarplot.add_coordinates()
+
+                    class PolarPlanePIradians(Scene):
+                        def construct(self):
+                            self.add(polarplot)
+
+            - ``"TAU radians"``: Fractional labels in the interval :math:`\\left[0, \\tau\\right]` (where :math:`\\tau = 2\\pi`) with :math:`\\tau` as a constant.
+
+              .. manim:: PolarPlaneTAUradians
+                    :ref_classes: PolarPlane
+                    :save_last_frame:
+
+                    polarplot = PolarPlane(azimuth_units="TAU radians").scale(0.4)
+                    polarplot.add_coordinates()
+
+                    class PolarPlaneTAUradians(Scene):
+                        def construct(self):
+                            self.add(polarplot)
+
+            - ``"degrees"``: Decimal labels in the interval :math:`\\left[0, 360\\right]` with a degree (:math:`^{\\circ}`) symbol.
+
+              .. manim:: PolarPlaneDegrees
+                    :ref_classes: PolarPlane
+                    :save_last_frame:
+
+                    polarplot = PolarPlane(azimuth_units="degrees").scale(0.4)
+                    polarplot.add_coordinates()
+
+                    class PolarPlaneDegrees(Scene):
+                        def construct(self):
+                            self.add(polarplot)
+
+            - ``"gradians"``: Decimal lables in the interval :math:`\\left[0, 400\\right]` with a superscript "g" (:math:`^{g}`).
+
+              .. manim:: PolarPlaneGradians
+                    :ref_classes: PolarPlane
+                    :save_last_frame:
+
+                    polarplot = PolarPlane(azimuth_units="gradians").scale(0.4)
+                    polarplot.add_coordinates()
+
+                    class PolarPlaneGradians(Scene):
+                        def construct(self):
+                            self.add(polarplot)
+
+    azimuth_offset : Optional[:class:`float`], optional
+        The angle offset of the azimuth labels, expressed in radians. If ``None`` is specified then it will
+        use the default specified by ``azimuth_units``.
+
+            - ``"PI radians"`` or ``"TAU radians"`` or ``None``: :math:`0`
+            - ``"degrees"`` or ``"gradians"``: :math:`\\frac{\\pi}{2}`
+
+    azimuth_direction : Optional[:class:`str`], optional
+        The direction of the azimuth labels.
+
+            - ``"CW"``: Clockwise.
+            - ``"CCW"`` Anti-clockwise.
+
+        If ``None`` is specified then it will use the default specified by ``azimuth_units``:
+
+            - `"PI radians"` or ``"TAU radians"`` or ``None``: ``"CW"``
+            - `"degrees"` or `"gradians"`: ``"CCW"``
```

Comment:
Again, I feel this inconsistency is somewhat artificial. By default, the direction should be mathematically positive, i.e., counterclockwise.

Reply:
I assume this is the same sentiment from the previous comment, so I'll make the change. Also, it appears I have made a docstring typo, radians should of course be CCW (not that it matters anymore since they will all be now).